Rakefile

```diff
@@ -12,10 +12,4 @@ Rake::TestTask.new("test") { |t|
   t.warning = true
 }
 
-require "gem_publisher"
-task :publish_gem do |t|
-  gem = GemPublisher.publish_if_updated("govspeak.gemspec", :rubygems)
-  puts "Published #{gem}" if gem
-end
-
-task :default => [:test]
+task default: [:test]
```

Comment:
It needs some more configuration for lint to be in there as well to load the task in.

I also don't think we have a defined pattern for rake default including linting although I do agree it's a good thing as we then have to override the test task which is an exceptional thing. Since this repo doesn't get much love I'm not too keen to take it down an exceptional route.